vendor/k8s.io/kubernetes/pkg/features/kube_features.go

```diff
@@ -577,7 +578,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 	AttachVolumeLimit:              {Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // remove in 1.19
 	CPUManager:                     {Default: true, PreRelease: featuregate.Beta},
 	CPUCFSQuotaPeriod:              {Default: false, PreRelease: featuregate.Alpha},
-	TopologyManager:                {Default: false, PreRelease: featuregate.Alpha},
+	TopologyManager:                {Default: true, PreRelease: featuregate.Beta},
```

Comment:
independent of this getting picked, we should update 4.5 feature set for latency sensitive clusters since this feature is on by default in 4.5/1.18.

see: https://github.com/openshift/api/blob/2cf49d05c5e272dbce4d65f4c5c1744000cb42e7/config/v1/types_feature.go#L106